client/src/chatCommands.ts

```diff
@@ -4,7 +4,8 @@ import globals from './globals';
 import * as modals from './modals';
 
 // Define a command handler map
-const chatCommands = new Map();
+type Callback = (...args: any) => void;
+const chatCommands = new Map<string, Callback>();
```

Comment:
so if we change the name of the other callback, then this would need to be ChatCallback or something right?

Reply:
This type is internal to this class, it's just because the linter gets stuck in a loop with 
new Map<string, (...args: any) => void;>();
It complains that there's a space, then if you remove it it complains it needs one. So i put a type alias in a few places.